.github/workflows/ci.yaml

```diff
@@ -90,6 +90,26 @@ jobs:
           path: ${{github.workspace}}/clar2wasm/src/standard/standard.wasm
           if-no-files-found: error
 
+  build:
+    name: Build binaries
+    runs-on: ubuntu-latest
+    needs: [build-test-artifacts]
```

Comment:
Why do we need the test artifacts to check the build ?

Reply:
we don't